src/openzeppelin/account.cairo

```diff
@@ -1,43 +1,52 @@
 use serde::Serde;
 use array::ArrayTrait;
-use array::SpanTrait;
 use starknet::ContractAddress;
-use starknet::contract_address::ContractAddressSerde;
 use openzeppelin::utils::check_gas;
+use openzeppelin::utils::span_to_array;
 
 const ERC165_ACCOUNT_ID: u32 = 0xa66bd575_u32;
 const ERC1271_VALIDATED: u32 = 0x1626ba7e_u32;
 
 const TRANSACTION_VERSION: felt252 = 1;
-const QUERY_VERSION: felt252 = 340282366920938463463374607431768211457; // 2**128 + TRANSACTION_VERSION
+const QUERY_VERSION: felt252 =
+    340282366920938463463374607431768211457; // 2**128 + TRANSACTION_VERSION
```

Comment:
moving the comment to a separate line would allow this to become a single line under the linter?

Reply:
Yes